src/Controller/FoodHeroController.php

```diff
@@ -126,12 +129,67 @@ public function showOffer(FoodHero $foodhero, Offer $offer)
      * @param Offer $offer
      * @return Response
      */
-    public function acceptOffer(FoodHero $foodhero, Offer $offer)
+    public function acceptOffer(FoodHero $foodhero, Offer $offer) : Response
     {
         $em = $this->getDoctrine()->getManager();
         $offer->setFoodhero($foodhero);
         $em->flush();
         
         return $this->redirectToRoute('foodhero_list_offers', ['id' => $foodhero->getId()]);
     }
+
+
+    /**
+     * @Route("/position/{latitude}/{longitude}")
+     * @param float $latitude
+     * @param float $longitude
+     * @param SessionInterface $session
+     * @return Response
+     */
+    public function showCoordinates(?float $latitude, ?float $longitude, SessionInterface $session)  : Response
+    {
+
+            $session->set('latitude', $latitude);
+            $session->set('longitude', $longitude);
+            return $this->json('');
```

Comment:
en fait je pense qu'on peut faire meme un return new Response(''); pas la peine d'un $this->json